packages/mdc-radio/_mixins.scss

```diff
@@ -53,6 +53,9 @@ $mdc-radio-ripple-target: ".mdc-radio__ripple";
     @include mdc-radio-unchecked-stroke-color($mdc-radio-unchecked-color, $query);
     @include mdc-radio-checked-stroke-color($mdc-radio-baseline-theme-color, $query);
     @include mdc-radio-ink-color($mdc-radio-baseline-theme-color, $query);
+    @include mdc-radio-disabled-unchecked-stroke-color($mdc-radio-circle-color, $query);
```

Comment:
Can we rename this variable to have `*-disabled-*` name?

Reply:
the current color for `$mdc-radio-circle-color` is on-surface, is this correct?

Reply:
`on-surface` is correct. However, what opacity should be is more ambiguous.

Reply:
The plan is to make this only about adding mixins, then correcting incorrect colors (mostly opacities) in separate PR so the changes are clear.